iso15118/evcc/comm_session_handler.py

```diff
@@ -429,6 +429,7 @@ async def start_comm_session(self, host: IPv6Address, port: int, is_tls: bool):
             self.iface,
             self.ev_controller,
         )
+        comm_session.is_tls = is_tls
```

Comment:
Just one comment - could you add a comment along the lines of "Overwriting is_tls with the setting returned from SDP response. Bear in mind the `is_tls` field in config still reflects what was provided in evcc_config."